bot/cogs/cyber.py

```diff
@@ -78,11 +82,25 @@ def __init__(self, bot: Bot):
         Gets information about a specific CyberStart Game level and challenge.
         If the date is before the start date of game (15th January 2019) it will redirect to game() instead
         """
-
         if datetime.date.today() < datetime.date(2019, 1, 15):
             await self.game.callback(self, ctx)
             return
 
+    async def on_message(self, message):
+        # Check the current command context
+        ctx = await self.bot.get_context(message)
+        # If message is a command, ignore regex responses.
+        if ctx.valid:
+            return
+        if self.game_level_regex.match(message.content):
+            level_num, challenge_num = int(match.group(2)), int(match.group(3))
```

Comment:
So `match` isn't defined here. `self.game_level_regex.match(message.content)` indeed returns a match, but it's not being saved in a variable. Try doing

```python
match = self.game_level_regex.match(message.content)
if match:
  # do stuff with the match
```

instead?

Reply:
omg thank u ily will do once the thingy finishes